lib/activerecord-import/import.rb

```diff
@@ -961,6 +964,11 @@ def import_associations(models, options)
       end
     end
 
+    def schema_columns_hash
+      load_schema unless schema_loaded?
+      @schema_columns_hash ||= connection.schema_cache.columns_hash(table_name)
+    end
```

Comment:
@leonidkroka @jkowens 

I think the `@schema_columns_hash` should be cleared by `.reset_column_information`.

👉 https://github.com/zdennis/activerecord-import/issues/809